Configuration-Interaction/CI_DL.py

```diff
@@ -117,20 +112,14 @@
 for CI_ITER in range(max_guess - 1):
 
     # Subspace Matrix, Gij = < bi | H | bj >
-    #for i in range(num_vecs - num_eig, num_vecs):
     for i in range(0, num_vecs):
         # Build sigma for each b
         ciwfn.sigma(cvecs, svecs, i, i)
```

Comment:
Ah, sorry you need to write before you can read. If you look up you can see that we write out blank lines to `dvecs`, you would need to do the same for `svecs`. Running the following snippet is fine:

```
H = np.array(ciwfn.hamiltonian(0))

# Setup test vecs
C1 = np.random.rand(H.shape[0])
C2 = np.random.rand(H.shape[0])

dvecs.np[:] = C1
dvecs.write(0, 1)
dvecs.symnormalize(1 / np.linalg.norm(C1), 0)
C1 = np.array(dvecs)

dvecs.np[:] = C2
dvecs.write(1, 0)
dvecs.symnormalize(1 / np.linalg.norm(C2), 1)
C2 = np.array(dvecs)

# Compute Explicit sigma
S1 = np.dot(H, C1)
S2 = np.dot(H, C2)
print(" ")

ciwfn.sigma(dvecs, svecs, 0, 0)
print(np.allclose(S1, svecs))

ciwfn.sigma(dvecs, svecs, 1, 1)
print(np.allclose(S2, svecs))
```

Is the excited state vector both normalized and symmetrized?